config/variables.go

```diff
@@ -0,0 +1,171 @@
+// Package config defines the default configuration and binds to environment variables
+package config
+
+import (
+	"github.com/flashbots/mev-boost-relay/common"
+	"github.com/spf13/viper"
+)
+
+const (
+	// Default values
+	DefaultNetwork     = ""
+	DefaultRedisURI    = "localhost:6379"
+	DefaultPostgresDSN = ""
+	DefaultLogJSON     = false
+	DefaultLogLevel    = "info"
+
+	APIDefaultListenAddr         = "localhost:9062"
+	APIDefaultBlockSim           = "http://localhost:8545"
+	APIDefaultSecretKey          = ""
+	APIDefaultLogTag             = ""
+	APIDefaultLogVersion         = false
+	APIDefaultPprofEnabled       = false
+	APIDefaultInternalAPIEnabled = false
+	APIDefaultDebug              = false
+
+	WebsiteDefaultListenAddr        = "localhost:9060"
+	WebsiteDefaultShowConfigDetails = false
+	WebsiteDefaultLinkBeaconchain   = "https://beaconcha.in"
+	WebsiteDefaultLinkEtherscan     = "https://etherscan.io"
+	WebsiteDefaultRelayURL          = ""
+	WebsiteDefaultPubkeyOverride    = ""
+
+	// Common: Api, HouseKeeper & Website
+	KeyNetwork        = "KeyNetwork"
+	KeyRedisURI       = "KeyRedisURI"
+	KeyPostgresDSN    = "KeyPostgresDSN"
+	KeyBeaconNodeURIs = "KeyBeaconNodeURIs"
+	KeyLogJSON        = "KeyLogJSON"
+	KeyLogLevel       = "KeyLogLevel"
+
+	// API cmd
+	KeyAPIListenAddr   = "KeyAPIListenAddr"
+	KeyAPIBlockSimURL  = "KeyAPIBlockSimURL"
+	KeyAPISecretKey    = "KeyAPISecretKey"
+	KeyAPILogTag       = "KeyAPILogTag"
+	KeyAPILogVersion   = "KeyAPILogVersion"
+	KeyAPIPprofEnabled = "KeyAPIPprofEnabled"
+	KeyAPIInternalAPI  = "KeyAPIInternalAPI"
+	KeyAPIDebug        = "KeyAPIDebug"
+
+	// API services
+	KeyBlockSimMaxConcurrent        = "KeyBlockSimMaxConcurrent"
+	KeyForceGetHeader204            = "KeyForceGetHeader204"
+	KeyDisableBlockPublishing       = "KeyDisableBlockPublishing"
+	KeyDisableLowprioBuilders       = "KeyDisableLowprioBuilders"
+	KeyNumActiveValidatorProcessors = "KeyNumActiveValidatorProcessors"
+	KeyNumValidatorRegProcessors    = "KeyNumValidatorRegProcessors"
+	KeyGetpayloadRetryTimeoutMs     = "KeyGetpayloadRetryTimeoutMs"
+
+	// Website
+	KeyWebsiteListenAddr        = "KeyWebsiteListenAddr"
+	KeyWebsiteShowConfigDetails = "KeyWebsiteShowConfigDetails"
+	KeyWebsiteLinkBeaconchain   = "KeyWebsiteLinkBeaconchain"
+	KeyWebsiteLinkEtherscan     = "KeyWebsiteLinkEtherscan"
+	KeyWebsiteRelayURL          = "KeyWebsiteRelayURL"
+	KeyWebsitePubkeyOverride    = "KeyWebsitePubkeyOverride"
+
+	// Database
+	KeyDBDontApplySchema = "KeyDBDontApplySchema"
+	KeyDBTablePrefix     = "KeyDBTablePrefix"
+	KeyDBRunTests        = "KeyKeyDBRunTests"
+	KeyDBTestDSN         = "KeyKeyDBTestDSN"
+
+	// Datastore
+	KeyDisableBidMemoryCache = "KeyDisableBidMemoryCache"
```

Comment:
`KeyDisableBidMemoryCache` is unused right now, I suppose it should be ?

Reply:
Good catch!

Reply:
The variable was removed in https://github.com/flashbots/mev-boost-relay/pull/206. I miss this during some rebase.